packages/pwa/app/components/_app-config/index.jsx

```diff
@@ -98,6 +98,7 @@ AppConfig.restore = (locals = {}) => {
 
     locals.api = new CommerceAPI({...apiConfig, locale, currency})
     locals.basePath = basePath
+    locals.locale = locale
```

Comment:
Note: this is certainly not the right approach to pass `locale` to the SDK. But to demonstrate the suggestion, it uses the same workaround from the spike PR.